Tests/SonarScanner.MSBuild.PreProcessor.Test/RoslynAnalyzerProviderTests.cs

```diff
@@ -137,7 +290,225 @@ public void RoslynConfig_ValidProfile()
         context.AssertInfoMessages("Provisioning analyzer assemblies for cs...");
         context.AssertCorrectRulesets();
         context.AssertExpectedAssemblies(@"c:\assembly1.dll", @"d:\foo\assembly2.dll");
-        context.AssertExpectedPluginsRequested(["wintellect", "csharp"]);
+        context.AssertExpectedPluginsRequested(
+            [new() { Key = "wintellect", Version = "1.13.0", StaticResourceName = "wintellect.zip" },
+            new() { Key = "csharp", Version = "1.42.0", StaticResourceName = "SonarAnalyzer.zip" }]);
+        context.AssertExpectedAdditionalFileExists(expectedSonarLintXml);
+    }
+
+    [TestMethod]
+    public void RoslynConfig_ValidProfile_Security_Frontend()
```

Comment:
I would not write 3 dedicated UT for security. Add the scaffolding to the existing UTs above